src/sentry/tagstore/snuba/backend.py

```diff
@@ -774,6 +803,28 @@ def get_perf_groups_user_counts(
             },
         )
 
+    def get_generic_groups_user_counts(
```

Comment:
Same comment regarding duplication - this is the same as `get_groups_user_counts` except for the dataset.